examples/hono/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": "1.0.0",
+  "name": "hono",
+  "main": "src/index.js",
+  "devDependencies": {
+    "bun-types": "^0.0.83"
+  },
+  "dependencies": {
+    "hono": "^1.6.4"
+  },
+  "scripts": {
+    "start": "cd src && bun run index.ts"
```

Comment:
Sorry for all the small nitpicks, but could you change this to ``bun run src/index.ts``?